README.md

```diff
@@ -76,10 +76,16 @@ This repo also includes related tools for [running tasks](#kubernetes-run) and [
 * Ruby 2.3+
 * Your cluster must be running Kubernetes v1.10.0 or higher<sup>1</sup>
 * Each app must have a deploy directory containing its Kubernetes templates (see [Templates](#using-templates-and-variables))
+* You must remove the` kubectl.kubernetes.io/last-applied-configuration` annotation from any resources in the namespace that are not included in your deploy directory. This annotation is added automatically when you create resources with `kubectl apply`. `kubernetes-deploy` will prune any resources that have this annotation and are not in the deploy directory.<sup>2,3</sup>
+* Each app managed by `kubernetes-deploy` must have its own exclusive Kubernetes namespace.
```

Comment:
I think you're adding this back accidentally because of the rebase. Please reconcile your changes to this file against  the ones made here: https://github.com/Shopify/kubernetes-deploy/pull/439/files#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
Fixed